src/components/Blogs/index.test.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { render } from "react-dom";
+import Blogs from "./index";
+import { act } from "react-dom/test-utils";
+
+let container;
+
+beforeEach(() => {
+  container = document.createElement("div");
+  document.body.appendChild(container);
+});
+
+afterEach(() => {
+  document.body.removeChild(container);
+  container = null;
+});
+
+describe("Blogs", () => {
+  it("Renders without crashing", async () => {
+    const mockData = [
+      {
+        title: {
+          rendered: "You can find safe place for yourself here",
+        },
+        jetpack_featured_media_url:
+          "https://safeplace102505649.files.wordpress.com/2020/11/womentogether.jpg",
+        id: 78,
+      },
+    ];
+
+    jest.spyOn(global, "fetch").mockImplementation(() =>
+      Promise.resolve({
+        json: () => Promise.resolve(mockData),
+      })
+    );
+
+    await act(async () => {
+      render(<Blogs />, container);
```

Comment:
the thing is, the example code i showed you doesn't use render. it uses render**er**
```
const tree = renderer.create(<MemberCard />).toJSON();
  expect(tree).toMatchSnapshot();
```

basically, i'm fairly sure you don't need to do this whole container thing. just literally copy the code above and put it in the async, change it to Blogs.

Reply:
like this? 
`describe("Blogs", () => {
  it("Renders without crashing", async () => {
    const mockData = [
      {
        title: {
          rendered: "You can find safe place for yourself here",
        },
        jetpack_featured_media_url:
          "https://safeplace102505649.files.wordpress.com/2020/11/womentogether.jpg",
        id: 78,
      },
    ];

    jest.spyOn(global, "fetch").mockImplementation(() =>
      Promise.resolve({
        json: () => Promise.resolve(mockData),
      })
    );

    await act(async () => {
      const tree = renderer.create(<Blogs />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
`

Reply:
```
      expect(tree).toMatchSnapshot();
```

this needs to be outside the act i believe

you are basically awaiting the fetch using act

and THEN you want to match the snapshot

Reply:
then tree will be undefined or should i declare it outside of function?

Reply:
I tried this before too. It doesn't work
``` let tree;
    await act(async () => {
      tree = renderer.create(<Blogs />).toJSON();
    });
    expect(tree).toMatchSnapshot(); 
```

Reply:
this is what I get
<img width="1440" alt="Screen Shot 2020-11-20 at 16 12 14" src="https://user-images.githubusercontent.com/67760504/99803921-47ad1880-2b4b-11eb-8f75-6031fb8766ca.png">